### PR TITLE
Add _candidates_en/SotaroKimura_2.md

### LINKED ISSUE #105: 英訳：木村宗太郎 さんによる「Spark Structured Streamingで行う統合データ処理」
ref: #101

### PR DESCRIPTION
**Pull request by richardimaoka**
Closes #105

### CODE REVIEW

File: _candidates_en/SotaroKimura_2.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Sotaro Kimura
+title: "Unified data processing with Spark Structured Streaming"
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: kimutansk
+github: kimutansk
+icon: https://s.gravatar.com/avatar/5d869b93035410ece41ff0d86d8c5c14
+organization: DWANGO Co., Ltd.
```
Comment (richardimaoka) [inline _candidates_en/SotaroKimura_2.md]: Confirmed the company name here http://dwango.co.jp/english/information/.

**Comment by richardimaoka** [comment]
@smroy can you please check this? thank you

File: _candidates_en/SotaroKimura_2.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+tags:
+  - Big Data / Fast Data
+  - Tools
+suggestions:
+  - People who want to know the difference between batch processing and stream processing
+  - People who want to understand concepts around stream processing 
+  - People who want to know practical uses cases of Spark Structured Streaming
```
Comment (smroy) [inline _candidates_en/SotaroKimura_2.md]: change to "use cases" instead of "uses cases"

File: _candidates_en/SotaroKimura_2.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+---
+Stream processing has come far beyond being an alternative to batch processing, and now it is considered as a unified data processing system.
```
Comment (smroy) [inline _candidates_en/SotaroKimura_2.md]: Consider changing to "+Stream processing has progressed far beyond being an alternative to batch processing and is now considered a unified data processing system."

File: _candidates_en/SotaroKimura_2.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+This session will talk about and review OSS stream processing systems like below,
```
Comment (smroy) [inline _candidates_en/SotaroKimura_2.md]: Consider changing to "+This session will describe and review OSS stream processing systems such as those listed below,"

File: _candidates_en/SotaroKimura_2.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+and will introduce Spark Structured Processing as a recent example of unified data processing system.
```
Comment (smroy) [inline _candidates_en/SotaroKimura_2.md]: Consider changing to "+and will introduce Spark Structured Processing as a recent example of an unified data processing system."

File: _candidates_en/SotaroKimura_2.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Lambda Architecture
+- Kappa Architecture
+- Unified Data Processing
+
+Finally, I will discuss issues we encountered upon running Spark Structured Streaming in production,
```
Comment (smroy) [inline _candidates_en/SotaroKimura_2.md]: Consider changing to "+Finally, I will discuss issues that we encountered while running Spark Structured Streaming in production,"

File: _candidates_en/SotaroKimura_2.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+how it feels like, and caveats you should keep in mind.
```
Comment (smroy) [inline _candidates_en/SotaroKimura_2.md]: Consider changing to "+what it felt like to run, and caveats that you should keep in mind."

**Comment by smroy** [comment]
@richardimaoka I have added suggested changes.

**Comment by richardimaoka** [comment]
Thanks @smroy. I reflected your comments, could you approve this now?

**Pull request merged**